webrtc.html

```diff
@@ -1448,8 +1455,11 @@ <h4>Set the RTCSessionDescription</h4>
                         to <code>null</code>. Set both
                         <var>connection</var>.<a>[[\LastCreatedOffer]]</a> and
                         <var>connection</var>.<a>[[\LastCreatedAnswer]]</a> to
-                        <code>""</code>. Finally set <var>connection</var>'s
-                        <a>signaling state</a> to <code>"stable"</code>.</p>
+                        <code>""</code>, and finally set <var>connection</var>'s
+                        <a>signaling state</a> to <code>"stable"</code>.
+                        If <var>connection</var>.<a>[[\CurrentLocalDescription]]</a>
+                        was previously <code>null</code>, <a>release early
+                          candidates</a>.</p>
```

Comment:
We could have early candidates after an ICE restart too, right? In that case [[CurrentLocalDescription]] wouldn't be null even if the candidates are early?
Can we always "release early candidates"? If there aren't any early candidates this would be a NO-OP anyway.

Reply:
@henbos good point! That should work. I think I was worried about rollback, but after the move that's not a problem anymore. I might also have been worried about SLD() failure, but at this point in the algorithm the description has already been successfully applied.